frontend/lang/messages/en-US.json

```diff
@@ -602,6 +602,12 @@
     "import-with-zip": "Import with .zip",
     "create-recipe-from-an-image": "Create Recipe from an Image",
     "create-recipe-from-an-image-description": "Create a recipe by uploading an image of it. Mealie will attempt to extract the text from the image using AI and create a recipe from it.",
+    "import-from-plaintext": "Import Recipe from Plain Text",
+    "create-recipe-from-text": "Create Recipe from Text",
+    "create-recipe-from-text-description": "Enter your recipe text below and we'll help you format it",
```

Comment:
I think something a little more verbose would work better; since Mealie is self-hosted, this is often times the only place users will ever see new features, and they may wonder "what's the point of this?"

Maybe:
```suggestion
    "create-recipe-from-text-description": "Type out your recipe in plain text from any source, and we'll send it to your OpenAI service to create a recipe from it. Be sure to include everything needed to create the recipe (the recipe name, description, all of the steps and ingredients, etc.). If your device supports it, you can even use text-to-speech and speak your recipe out loud.",
```

Reply:
yes, very good point.